synapse/lib/storm.py

```diff
@@ -488,18 +501,56 @@ def cmpr(tufo):
 
     def _cmprCtorTag(self, oper):
         tag = self._reqOperArg(oper,'valu')
-
-        props = {}
-        def cmpr(tufo):
+        glob_props = collections.defaultdict(set)
+        reg_props = {}
+
+        if tag.startswith(self._rt_glob_smmark) or self._rt_glob_smark in tag or self._rt_glob_mmark in tag:
```

Comment:
might be worth testing, but:
```
str_x in str_y:
```
involves a brute force sliding comparison which we could probably avoid (a couple times) with a single split and doing equality comparison.
```
parts = tag.split('.')
if '*' in parts or '**' in parts:
    globlogic()
```

likely faster/simpler/clearer even if we dont use parts for anything but deciding we're in the regex case